storage/SMAIndexSubBlock.cpp

```diff
@@ -488,6 +490,12 @@ std::size_t SMAIndexSubBlock::EstimateBytesPerTuple(
   return 1;
 }
 
+std::size_t SMAIndexSubBlock::EstimateBytesPerBlock(
+    const CatalogRelationSchema &relation,
+    const IndexSubBlockDescription &description) {
+  return 0;
```

Comment:
Guess I gotta go make the SMA better now.. That's been on my TODO for a while, ty for adding this.